src/Assets/Config.php

```diff
@@ -212,8 +214,8 @@ protected static function normalize_path( string $path ): string {
 			$plugins_content_dir_position === false
 			&& $themes_content_dir_position === false
 		) {
-			// Default to plugins.
-			$path = $plugin_dir . $path;
+			// Default to path.
+			$path = $path;
```

Comment:
I think this change would mean that any attempt to use `::set_path( $some_path )` would result in absolute paths being used rather than relative ones.

Perhaps we should determine if `$path` is absolute, and if so do `$path = $path;`, otherwise retain `$path = $plugin_dir . $path` as the default.

Reply:
I think that is already the case after all. Addedf a testcase `should_set_root_path_correctly` to show what we are giving what we are receiving in its case and i think that makes sense.

If the assets library was provider by a relative path, how would it know how to translate it for all the checks it needs to perform. What comes to mind first is the check of a minified version.

Reply:
Here's [a follow-up PR](https://github.com/stellarwp/assets/pull/26) that implements what I am referring to. I think one of the confusions is the distinction between `Config::set_path()` and `Asset::set_path()`